### ISSUE TITLE
Robustness improvment for 3DText

### ISSUE THREAD

**Issue by Nedok** [Indimeco/QRPG#41]
If a user accidentally remove a 3DText object form there character sheet, that sheet becomes impossible to save. This is quite easy mistake to do.

This problem could be reduced in severity if the last save data (of the missing 3DText) could be rewritten to the save file when closing the book.

Demonstration of problem: [http://imgur.com/y1gbPUc](http://imgur.com/y1gbPUc)
Error output: `Error calling Lua function saveCharacter `

**Comment by Indimeco** [comment]
This issue runs a little deeper than I first thought. I'll continue looking into it but it may be a while before I get around.

**Comment by Nedok** [comment]
An alternative is to allow the use for the new 3D UI for text input.

**Comment by Indimeco** [comment]
This is true. I haven't got a chance to experiment with the new 3D UI features, but it seems like it could be added as an additional table type to the character sheet source and generated by the char core along with the other buttons.
